lib/src/remote_frame_buffer_widget.dart

```diff
@@ -36,11 +39,17 @@ class RemoteFrameBufferWidget extends StatefulWidget {
     final void Function(Object error)? onError,
     final String? password,
     final int port = 5900,
+    final ValueNotifier<Map<String, dynamic>>? keyEventNotifier,
```

Comment:
question: is it a usecase that you need right now? Knowing from outside of this widget, which keys are being pressed? I'd expect that you would just add a surrounding `RawKeyboardListener` widget (https://api.flutter.dev/flutter/widgets/RawKeyboardListener-class.html) to achieve this.